includes/discovery/route.inc.php

```diff
@@ -61,6 +65,61 @@
     include Config::get('install_dir') . "/includes/discovery/route/{$device['os']}.inc.php";
 }
 
+Log::info('IPV6-MIB::ipv6RouteTable');
+$oids = SnmpQuery::hideMib()->walk('IPV6-MIB::ipv6RouteTable')->table(3) ?? [];
```

Comment:
table() never returns null

```suggestion
$oids = SnmpQuery::hideMib()->walk('IPV6-MIB::ipv6RouteTable')->table(3);
```